diff -U2 mod_nss-1.0.7/nss.conf.in mod_nss-1.0.7/nss.conf.in
--- mod_nss-1.0.7/nss.conf.in	2008-06-17 09:14:46.944230209 +0300
+++ mod_nss-1.0.7/nss.conf.in	2008-06-17 09:37:06.875135679 +0300
@@ -1,3 +1,4 @@
-#
+LoadModule nss_module	modules/libmodnss.so
+
 # This is the Apache server configuration file providing SSL support using.
 # the mod_nss plugin.  It contains the configuration directives to instruct
@@ -9,4 +10,5 @@
 #
 
+<IfModule mod_nss.c>
 #
 # When we also provide SSL we have to listen to the 
@@ -15,5 +17,5 @@
 # Note: Configurations that use IPv6 but not IPv4-mapped addresses need two
-#       Listen directives: "Listen [::]:443" and "Listen 0.0.0.0:443"
+#       Listen directives: "Listen [::]:8443" and "Listen 0.0.0.0:8443"
 #
-Listen 443
+Listen 8443
 
@@ -69,15 +71,15 @@
 ##
 
-<VirtualHost _default_:443>
+<VirtualHost _default_:8443>
 
 #   General setup for the virtual host
-#DocumentRoot "@apache_prefix@/htdocs"
-#ServerName www.example.com:443
+#DocumentRoot "/home/services/httpd/html"
+#ServerName www.example.com:8443
 #ServerAdmin you@example.com
 
 # mod_nss can log to separate log files, you can choose to do that if you'd like
 # LogLevel is not inherited from httpd.conf.
-#ErrorLog @apache_prefix@/logs/error_log
-#TransferLog @apache_prefix@/logs/access_log
+#ErrorLog logs/error_log
+#TransferLog logs/access_log
 LogLevel warn
 
@@ -114,5 +116,5 @@
 #   keys. The database consists of 3 files: cert8.db, key3.db and secmod.db.
 #   Provide the directory that these files exist.
-NSSCertificateDatabase @apache_conf@
+NSSCertificateDatabase @apache_conf@/nss
 
 #   Database Prefix:
@@ -190,5 +192,5 @@
     NSSOptions +StdEnvVars
 </Files>
-<Directory "@apache_prefix@/cgi-bin">
+<Directory "/home/services/httpd/cgi-bin">
     NSSOptions +StdEnvVars
 </Directory>
@@ -203,0 +206 @@
+</IfModule>
